crates/figma_import/src/transform_flexbox.rs

```diff
@@ -121,7 +122,10 @@ fn compute_layout(
 
     // Frames can implement Auto Layout on their children.
     if let Some(frame) = node.frame() {
-        style.layout_style.position_type = PositionType::Relative;
+        style.layout_style.position_type = match frame.layout_positioning {
+            LayoutPositioning::Absolute => PositionType::Absolute,
+            LayoutPositioning::Auto => PositionType::Relative,
+        };
```

Comment:
Wow nice, so this doesn't require any changes to how it is passed into taffy? I guess setting this to absolute converts to the same type of attribute in taffy::style, which it supports?

Reply:
Yeah that's right. We already have absolute positioning supported in non auto layout cases, so just need to update the tranform_flexbox.rs. Turned out straight forward.